FortuneCookieModelGloveEmbedding.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# <a href="https://colab.research.google.com/github/WomenInDataScience-Seattle/FortuneCookie/blob/master/FortuneCookieModel.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+
+
+from tensorflow.python.keras.preprocessing.sequence import pad_sequences
+from tensorflow.python.keras.preprocessing.text import Tokenizer
+from tensorflow.python.keras.optimizers import Adam
+from tensorflow.python.keras.layers import Dense, GRU, Embedding, LSTM, Dropout
+from tensorflow.python.keras.models import Sequential
+from scipy.spatial.distance import cdist
+import tensorflow as tf
+from io import StringIO
+
+import numpy as np
+import pandas as pd
+import requests
+import os
+
+from library.common import get_sequence_of_tokens, generate_embeddings_index, get_word_index, generate_embedding_matrix, generate_padded_sequences, create_model_glove_embedding, generate_text
+# Glove embedding layer constants
+BASE_DIR = './training_data/'
+GLOVE_DIR = os.path.join(BASE_DIR, '')
+TEXT_DATA_DIR = os.path.join(BASE_DIR, '20_newsgroup')
+
```

Comment:
Eventually we are going to move these global params into _init_.py or library/common, and import them here. Thus, we can change them in one centralized place.

Reply:
Are we going to continue to refine our `GloVe` version of the fortune cookie? I would suspect we would not necessarily end up with a common global env variables. I suppose if we are saying global to the `GloVe` portion of the project it makes sense to put them in a separate file.

Reply:
Yeah - I kinda feel like because we are deliberately setting this repo up with 3 different models, it's confusing to have global variables that only apply to one model. I think there's an argument to leave them how they are.